### PR TITLE
Add an image missing check

### PR DESCRIPTION
**Pull request by karpa4o4**

### CODE REVIEW

File: bin/server.py [hunk 1]
```diff
@@ -57,6 +72,11 @@ def __init__(self, application, request, **kwargs):
         self._script_p = None
         self._terminal = None
 
+    def _init_mongodb(self):
+        self.logger.debug(options.db_name)
+        client = MongoClient(options.mongodb_host, options.mongodb_port)
+        self.db = client[options.db_name]
```
Comment (eugulixes) [inline bin/server.py]: You never use `self.db`, so remove the whole `_init_mongodb`.

File: bin/server.py [hunk 2]
```diff
@@ -86,6 +106,19 @@ def destroy(self):
 
     @remote
     def start(self, request, image_name, rows=24, cols=80):
+        self._init_mongodb()
+        user = User.objects.get(id=self.user_id)
+        firmwares = Firmware.objects.filter(user=user) \
+                                    .filter(status=Firmware.DONE) \
+                                    .order_by('-started_at')
```
Comment (eugulixes) [inline bin/server.py]: Why do you sort the records?

File: bin/server.py [hunk 3]
```diff
@@ -86,6 +106,19 @@ def destroy(self):
...
+        missing_status = True
+        for firmware in firmwares:
+            if firmware.name == image_name:
+                missing_status = False
+        
+        if (missing_status):
```
Comment (eugulixes) [inline bin/server.py]: The brackets are redundant.

**Comment by karpa4o4** [comment]
The task is irrelevant.

**Pull request closed without merging**